pyproject.toml

```diff
@@ -13,14 +13,8 @@ python = "^3.8"
 uhashring = "^2.1"
 marisa-trie = "^1.0.0"
 
-[tool.poetry.dev-dependencies]
-flake8 = "^5"
-flake8-annotations = "^2.9"
-flake8-bandit = "^4.1"
-flake8-black = "^0.3"
-flake8-bugbear = "^22"
-flake8-docstrings = "^1.6"
-mypy = "^0"
+[tool.poetry.group.extras.dependencies]
+prometheus-client = "^0.17.1"
```

Comment:
Will this pull in `prometheus-client` if someone adds the memcache cllient as a dependency? It would be nice if it's not a hard requirement.

Reply:
It is not a hard requirement, that is why I put it in the "extras" group. It won't be listed in package deps. :)